lib/stats_reporter.rb

```diff
@@ -61,22 +61,30 @@ def workflow_xml
   def waiting_count
     Dor::WorkflowService.count_objects_in_step(ingest_wf, 'start-ingest',
                                                'waiting', repository)
+  rescue Dor::WorkflowException => exception
```

Comment:
The repeated code added to this file brings me no joy but a naïve refactoring to force all of these `#*_count` methods to call another method whose job it is to do a thing and rescue `Dor::WorkflowException` seemed worse than some repetition because the new method could need too much knowledge of the different ways to call and handle methods within `Dor::WorkflowService`.